pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -911,6 +885,32 @@ class ShapePropagator : public PropertyPropBase {
             "aten::alias(Tensor self) -> Tensor",
             "aten::zero_(Tensor self) -> Tensor",
             "aten::tanh_backward(Tensor grad_output, Tensor output) -> Tensor",
+#if PYTORCH_VERSION_GE(1, 7)
+            "aten::logit(Tensor self, float? eps=None) -> Tensor",
+#endif
+#if PYTORCH_VERSION_GE(1, 9)
+            "aten::relu6(Tensor self) -> Tensor",
+            "aten::relu6_(Tensor self) -> Tensor",
+#endif
+#if PYTORCH_VERSION_GE(1, 11)
+            "aten::div(Tensor self, Tensor other, *, str? rounding_mode) -> Tensor",
+            "aten::div_(Tensor self, Tensor other, *, str? rounding_mode) -> Tensor",
+#endif
+#if PYTORCH_VERSION_GE(1, 11) && PYTORCH_VERSION_GE(1, 12)
+            "aten::slice_scatter(Tensor self, Tensor src, int dim=0, int? start=None, int? end=None, int step=1) -> Tensor",
+#endif
+#if PYTORCH_VERSION_GE(1, 12)
+            "aten::pin_memory(Tensor(a) self, Device? device=None) -> Tensor(a)",
+            "aten::gelu(Tensor self, *, str approximate='none') -> Tensor",
+            "aten::gelu_backward(Tensor grad_output, Tensor self, *, str approximate='none') -> Tensor",
+            "aten::native_dropout_backward(Tensor grad_output, Tensor mask, float scale) -> Tensor",
+#else
+            "aten::gelu(Tensor self) -> Tensor",
+#endif
+#if PYTORCH_VERSION_GE(1, 13)
```

Comment:
Nit: Update the shape analysis incremental, and do not modify the original code as possible.